packages/playwright-test/src/plugins/index.ts

```diff
@@ -14,3 +14,4 @@
  * limitations under the License.
  */
 export { webServer } from './webServerPlugin';
+export { vcs } from './vcsPlugin';
```

Comment:
I don't think your plugin helps those that are not using Git much. How about call it GitCommitInfo (or GitCommitInfo)?